openhtf/io/frontend/__main__.py

```diff
@@ -133,12 +200,21 @@ def main(argv):
     print('%s\nUsage: %s ARGS\n%s' % (e, sys.argv[0], FLAGS))
     sys.exit(1)
 
-  if not os.path.isdir(FLAGS.rundir):
-    print('ERROR: OpenHTF run directory does not exist', FLAGS.rundir,
-          file=sys.stderr)
-    sys.exit(1)
+  logs.setup_logger()
+
+  frontend_server = HTTPServer(WSGIContainer(app))
```

Comment:
I'm curious about the motivation behind Tornado versus flask's builtin webserver or some other one?